docs/visual-basic/language-reference/error-messages/elementname-is-obsolete-visual-basic-warning.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "'<elementname>' is obsolete (Visual Basic Warning)"
+title: "'<elementname>' is obsolete - Warning"
```

Comment:
It is probably okay not to even add the hyphen.

Reply:
There are multiple occurrences on this one. I'll get time soon to remove the hyphen.